h2o-automl/src/main/java/water/automl/api/schemas3/AutoMLV99.java

```diff
@@ -106,7 +107,7 @@ public AutoMLKeyV3(Key<AutoML> key) {
 
     Leaderboard lb = autoML.leaderboard();
     if (null == lb) {
-      lb = new Leaderboard(project_name, eventLog, autoML.getLeaderboardFrame(), sort_metric);
+      lb = new Leaderboard(project_name, eventLog.asLogger(EventLogEntry.Stage.ModelTraining), autoML.getLeaderboardFrame(), sort_metric, Leaderboard.ScoreData.auto);
```

Comment:
or I guess it's because we just want a dummy LB here, and we don't want to put it in DKV...
Still not nice though... even if I much prefer to have `DKV.put` in factory rather than in the constructor itself.
...
ok, let's leave it this way.